### ISSUE TITLE
Pathvisio 4 vs 3.3.0 DOI ref wrong address

### ISSUE THREAD

**Issue by ondrej-kuda** [PathVisio/pathvisio#204]
Hi,
I edited a pathway in PV4 and inserted refs as DOIs. When I open it in PV3 and upload it to web, the link is broken - an extra prefix https://doi.org/ is used.
Best regards,
Ondrej

![image](https://github.com/PathVisio/pathvisio/assets/111736085/7a944d9c-48f0-4609-9cc0-9d07d9902b9e)

**Comment by khanspers** [comment]
@mkutmon @egonw : I assume these references need to be adjusted in PV3 to upload correctly?

**Comment by egonw** [comment]
I can fix that

**Comment by egonw** [comment]
hi @ondrej-kuda, the DOI is in some way not clearly defined and the display guidelines often write something like "the DOI should be displayed as https://doi.org/10....", but in GPML it is just the unique part:

```xml
    <bp:PublicationXref xmlns:bp="http://www.biopax.org/release/biopax-level3.owl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" rdf:id="dc64d">
      <bp:ID rdf:datatype="http://www.w3.org/2001/XMLSchema#string">10.1074/jbc.M403484200</bp:ID>
      <bp:DB rdf:datatype="http://www.w3.org/2001/XMLSchema#string">DOI</bp:DB>
      <bp:TITLE rdf:datatype="http://www.w3.org/2001/XMLSchema#string"></bp:TITLE>
      <bp:SOURCE rdf:datatype="http://www.w3.org/2001/XMLSchema#string"></bp:SOURCE>
      <bp:YEAR rdf:datatype="http://www.w3.org/2001/XMLSchema#string"></bp:YEAR>
    </bp:PublicationXref>
```

Mind you, on the `classic` website, the title, authors, year, source, will not show up, but on the new website it will look up that info and still display it.

**Comment by ondrej-kuda** [comment]
Hi Egon,
thank you for your help. I used the GUI tool to insert the refs: Citation > New citation > identifier in DOI database > Query > Done.
Best regards,
Ondrej

**Comment by egonw** [comment]
Yes, that is the right way. The DOI support is still under development, and while it only provides limited support, I'm happy we support them in the first place, because not all literature is indexed in PubMed.
